packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -575,16 +582,23 @@ async function unlock (transfer) {
   const borshProof = borshifyOutcomeProof(last(transfer.proofs))
   const nearOnEthClientBlockHeight = new BN(last(transfer.nearOnEthClientBlockHeights))
 
+  // If this tx is dropped and replaced, lower the search boundary
+  // in case there was a reorg.
+  const safeHeightBeforeEthTx = await web3.eth.getBlockNumber() - 20
```

Comment:
Is this the same as something like `pendingUnlockTx.block - 20`?

Reply:
`pendingUnlockTx` will not have a `block` yet, since the transaction has not yet been mined